packages/elements/src/components/ui/banners/banner/Banner.tsx

```diff
@@ -49,6 +48,8 @@ const iconPerVariant: Record<BannerVariant, IconDefinition | undefined> = {
   error: stenaExclamationTriangle,
 };
 
+const leftContentWidth = "56px";
```

Comment:
If possible, do it with CSS properties. And use `calc(var(--swui-metric-indent) * 7)`, then it will adapt when changing the theme.

Reply:
No, this was changed because I preferred to have the padding left and right instead of only padding right + special logic to handle the indent on the left side. So design with this.